pyproject.toml

```diff
@@ -1,23 +1,22 @@
 [project]
 python = "^3.8"
-
-
-[tool.poetry]
 name = "e-referrals-service-api"
 version = "0.0.1-alpha"
 description = "TODO"
-package-mode = false
-
 license = "MIT"
 
 authors = [
 ]
 
 readme = 'README.md'
-
 repository = "https://github.com/NHSDigital/e-referrals-service-api"
-
 keywords = ["healthcare", "uk", "nhs"] #TODO add additional keywords
+dynamic = ["dependencies", "requires-python"]
```

Comment:
Is this to allow poetry to retrieve both the python version and main dependencies for the project from the `tool.poetry` section?

Reply:
Yes - I believe so.
I'll do some more investigation - but it was the suggested fix from the 'poetry check' output